### PR TITLE
[Testing] Added UI Test for manual test D6

### LINKED ISSUE #18726: [Testing] Create UI Test for manual test D6
### Description

Create UI Test for manual test D6.

```
<ScrollView>
    <VerticalStackLayout>
        <Label Text="1. Enter the string 'hi my name is james. nice to meet you.' exactly (case-sensitive)." />
        <Label Text="2. The test fails if the editor does not display 'Hi my name is james. Nice to meet you.' exactly (case-sensitive)." />
        <Editor>
            <Editor.Keyboard>
                <Keyboard x:FactoryMethod="Create">
                    <x:Arguments>
                        <KeyboardFlags>CapitalizeSentence</KeyboardFlags>
                    </x:Arguments>
                </Keyboard>
            </Editor.Keyboard>
        </Editor>

        <Label Text="1. Enter the string 'hi my name is james. nice to meet you.' exactly (case-sensitive)." />
        <Label Text="2. The test fails if the editor does not display 'Hi My Name Is James. Nice To Meet You.' exactly (case-sensitive)." />
        <Editor>
            <Editor.Keyboard>
                <Keyboard x:FactoryMethod="Create">
                    <x:Arguments>
                        <KeyboardFlags>CapitalizeWord</KeyboardFlags>
                    </x:Arguments>
                </Keyboard>
            </Editor.Keyboard>
        </Editor>

        <Label Text="1. Enter the string 'hi my name is james. nice to meet you.' exactly (case-sensitive)." />
        <Label Text="2. The test fails if the editor does not display 'HI MY NAME IS JAMES. NICE TO MEET YOU.' exactly (case-sensitive)." />
        <Editor>
            <Editor.Keyboard>
                <Keyboard x:FactoryMethod="Create">
                    <x:Arguments>
                        <KeyboardFlags>CapitalizeCharacter</KeyboardFlags>
                    </x:Arguments>
                </Keyboard>
            </Editor.Keyboard>
        </Editor>

        <Label Text="1. Enter the string 'hi my name is james. nice to meet you.' exactly (case-sensitive)." />
        <Label Text="2. The test fails if the editor does not display 'hi my name is james. nice to meet you.' exactly (case-sensitive)." />
        <Editor>
            <Editor.Keyboard>
                <Keyboard x:FactoryMethod="Create">
                    <x:Arguments>
                        <KeyboardFlags>CapitalizeNone</KeyboardFlags>
                    </x:Arguments>
                </Keyboard>
            </Editor.Keyboard>
        </Editor>

        <Label Text="1. Enter the string 'qweqw hi hi' exactly (case-sensitive)." />
        <Label Text="2. The test fails if there is no red underline under the string 'qweqw' to indicate a spelling error." />
        <Editor>
            <Editor.Keyboard>
                <Keyboard x:FactoryMethod="Create">
                    <x:Arguments>
                        <KeyboardFlags>Spellcheck</KeyboardFlags>
                    </x:Arguments>
                </Keyboard>
            </Editor.Keyboard>
        </Editor>
    </VerticalStackLayout>
</ScrollView>
```

### Steps to Reproduce

_No response_

### Link to public reproduction project repository

_No response_

### Version with bug

8.0.0-rc.2.9511

### Is this a regression from previous behavior?

No, this is something new

### Last version that worked well

Unknown/Other

### Affected platforms

iOS, Android, Windows, macOS

### Affected platform versions

_No response_

### Did you find any workaround?

_No response_

### Relevant log output

_No response_

### PR DESCRIPTION
**Pull request by jsuarezruiz**
### Description of Change

Added UI Test for manual test D6.

### Issues Fixed

Fixes #18726

### CODE REVIEW

File: src/Controls/tests/UITests/Tests/Issues/Issue18726.cs
Comment (jsuarezruiz) [inline src/Controls/tests/UITests/Tests/Issues/Issue18726.cs:18]: The Keyboard property is not working correctly on any platform. 

image
image

**Comment by rmarinho** [comment]
Is this something @tj-devel709  should take a look ? before we merge it.  I think think we should merge this if it's being ignored no?

File: src/Controls/tests/UITests/Tests/Issues/Issue18726.cs
Comment (PureWeen) [reply src/Controls/tests/UITests/Tests/Issues/Issue18726.cs:18]: Should we close this PR and log an issue?

**Review by PureWeen — CHANGES_REQUESTED** [review]

File: src/Controls/tests/UITests/Tests/Issues/Issue18726.cs
Comment (jsuarezruiz) [reply src/Controls/tests/UITests/Tests/Issues/Issue18726.cs:18]: We already have https://github.com/dotnet/maui/issues/17784
And this ignored tests could be useful when someone starts working on that issue.

**Comment by jsuarezruiz** [comment]
> Is this something @tj-devel709 should take a look ? before we merge it. I think think we should merge this if it's being ignored no?

That is a good question. Do we not include new tests that do not pass? Or do we include them, ignoring them and adding a reference both in the test to issue and in the issue to the test?. I think I prefer the second option. It can add value and facilitate the work on the fix. Although, for now, effectively being ignored does not prevent to continue passing the manual test.

**Comment by jfversluis** [comment]
I'm kinda fine either way, but we have the code here and linked to the issue that we need to fix. Let's close this for now and we'll add the test code whenever we get the actual fix for this feature as well.

**Pull request closed without merging**
